plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -148,20 +175,21 @@ var FbListInlineEdit = new Class({
 			}
 			if (typeOf(this.td.getNext()) === 'element') {
 				e.stop();
-				this.select(e, this.td.getNext());
+				this.select(e, this.getNextEditable(this.td));
 			}
 			break;
 		case 9:
-			//tab - don't navigate with tab - moofs form field tab ordering if we do
-			if (this.inedit) {
-				if (this.options.tabSave) {
-					if (typeOf(this.editing) === 'element') {
-						this.save(e, this.editing);
-					} else {
-						this.edit(e, this.td);
-					}
+			if (this.inedit && this.options.tabSave) {
+				if (typeOf(this.editing) === 'element') {
```

Comment:
I had to use mootools typeOf was because of the ambiguity with javascript typeof (would always return 'object')
As explained here... [https://davidwalsh.name/mootools-typeof](url)